portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeBodyScripts.vm

```diff
@@ -130,22 +130,22 @@
       dollarVersion = window.${d}.fn.jquery;
     } else {
       dollarVersion = 'not jQuery';
-      console.debug('Dollar is defined but is not jQuery');
-      console.debug($d);
+      console.log('Dollar is defined but is not jQuery');
```

Comment:
Why make these into log statements? Normal users will not understand these outputs - they are only for devs. I think that is why they were debug in the first place.